plugin/symbols.py

```diff
@@ -170,30 +197,27 @@ def process_document_symbol_recursive(self, quick_panel_items: List[sublime.Quic
                 st_details = "{} | {}".format(st_details, formatted_names)
             else:
                 st_details = formatted_names
+            tags = item.get("tags") or []
+            if SymbolTag.Deprecated in tags:
+                st_display_type = "⚠ {} - Deprecated".format(st_display_type)
             quick_panel_items.append(
                 sublime.QuickPanelItem(
                     trigger=name,
                     details=st_details,
                     annotation=st_display_type,
                     kind=(st_kind, st_icon, st_display_type)))
             children = item.get('children') or []
+            children = cast(List[DocumentSymbol], children)
```

Comment:
Ideally I shouldn't have used cast here,
but because mypy doesn't support recursive types, 
I couldn't typehint DocumentSymbol['children'] to be the type of `DocumentSymbol`.
https://github.com/sublimelsp/LSP/pull/1541/files#diff-049653877a86e4c220cd537d174655150e80b4ce0077d48742cc72b4654e2112R135

Reply:
I've removed this line and that didn't seem to break either mypy or pyright.
At the same time, this line doesn't seem to help pyright at least in understanding the type of `children` (still shows as `List[Unknown]`.

What seems to work for pyright is using the type comment (`# type: List[DocumentSymbol]`).

Reply:
You are right, 

I swear I saw an error :D but apparently the error is no longer there 

I will add the `# type: List[DocumentSymbol]`

Reply:
16ebb7e68cd895a92922dc92444236aeaf76c0bd